### ISSUE TITLE
Suggestion: looping

### ISSUE THREAD

**Issue by aekanman** [iam4x/pokemongo-webspoof#42]
given 2 or more locations, it would be nice to have the location changing between these locations, and looping continuously, until the user stops.

it would be fantastic for moving back and forth between couple of lure locations. :purple_heart:

**Comment by DannyPeo** [comment]
You could do this manually using the map? The problem would be if the locations were far apart the user would incur a soft ban meaning they couldn't do anything for up to half an hour.

**Comment by aekanman** [comment]
Yes, you could do this manually but automating this in a short radius saves the user a lot of arrow clicks.

**Comment by muescha** [comment]
it is also interesting to get miles for the eggs

**Comment by muescha** [comment]
crete some walkink path and lets stop for some time at a location
